src/Settings.php

```diff
@@ -251,6 +251,16 @@ public function add_options_page_metabox() {
 			]
 		);
 
+		$cmb->add_field(
+			[
+				'name' => __( 'SimpleShop API Endpoint URL', 'simpleshop-cz' ),
+				'desc' => __( '[SERVICE FLAG] You can here override URL to SimpleShop API. Leave blank to use default API.', 'simpleshop-cz' ),
+				'id'   => 'ssc_api_endpoint_url',
+				'type' => 'text',
+				'classes_cb' => [ $this, 'show_endpoint_url_when_flagged' ],
```

Comment:
[CMB2](https://github.com/CMB2/CMB2), plugin, který používáme pro metaboxy a nastavení, má pro `add_field` parametr `show_on_cb`, kterým se dá nastavit, kdy se pole má zobrazovat. Metoda má vracet `true/false`, ukázka toho jak to použít je [zde](https://github.com/CMB2/CMB2/wiki/Adding-your-own-show_on-filters#example-using-the-show_on_cb-to-limit-the-display-of-a-metabox-unless-metadata-exists). Použil bych tedy asi to..:-)

Reply:
OK, založil jsem na to #33, ale v tomto PR to nechám, aby tomu byla zachována konzistence se zbytkem dosavadního kódu v pluginu.